emulator/src/extensions/emu_args.c

```diff
@@ -156,11 +164,14 @@ static const optDoc_t argDocs[] =
     { 0,  argFuzzButtons, "y|n",   "Set whether buttons are fuzzed" },
     { 0,  argFuzzTouch,   "y|n",   "Set whether touchpad inputs are fuzzed" },
     { 0,  argFuzzMotion,  "y|n",   "Set whether motion inputs are fuzzed" },
+    { 0,  argHeadless,    NULL,    "Runs the emulator without a window." },
     { 0,  argHideLeds,    NULL,    "Don't draw simulated LEDs next to the display" },
     {'l', argLock,        NULL,    "Lock the emulator in the start mode" },
     {'m', argMode,        "MODE",  "Start the emulator in the swadge mode MODE instead of the main menu"},
     { 0,  argModeSwitch,  "TIME",  "Enable or set the timer to switch modes automatically" },
     { 0,  argModeList,    NULL,    "Print out a list of all possible values for MODE" },
+    {'p', argPlayback,    "FILE",  "Play back recorded emulator inputs from a file" },
+    {'r', argRecord,      "FILE",  "Record emulator inputs to a file" },
```

Comment:
I don't love having two separate arrays that have to stay in sync, but this is still pretty nice.